scripts/jenkins/ardana/manual/lib.sh

```diff
@@ -196,10 +194,10 @@ function deploy_ses_vcloud {
 }
 
 function bootstrap_nodes {
-  if is_crowbar; then
-        ansible_playbook bootstrap-crowbar-nodes.yml
+  if [ "$(get_cloud_product)" == "crowbar" ]; then
```

Comment:
Could you use `[[` to stay consistent with the rest of the code?

Reply:
done